### ISSUE TITLE
search_quotes + retrieve_historical_data does not work anymore

### ISSUE THREAD

**Issue by tavasti** [alvarobartt/investpy#338]
For some reason search_quotes and retrieve_historical_data with that does not work anymore.

```
>>> import investpy
>>> df = investpy.search_quotes(text='IE00BFNM3P36',countries=["germany"])
>>> df[0].__dict__['tag']
'/etfs/saem?cid=1162546'
>>> df[0].__dict__['name']
'iShares MSCI EM IMI ESG Screened UCITS ETF USD (Acc)'
>>> data = df[0].retrieve_historical_data(from_date='01/01/2020',to_date='01/01/2021')
>>> print(data)
None
```

My investpy is version 1.0.5

**Comment by alvarobartt** [comment]
Hi @tavasti, this is an issue already spotted in #333 and solved by @danilogalisteu in #329, I'll merge this ASAP so as to release another bug-fix version in order to solve this breaking issue! Sorry for the inconvenience, stay tuned for more updates :hugs:

Next release will be `investpy v1.0.6` :fire:

**Comment by alvarobartt** [comment]
You can now install the latest `investpy` release as it follows:

```bash
pip install investpy==1.0.6
```

Thanks for the support! :muscle::fire:
